src/extension.ts

```diff
@@ -1460,7 +1465,82 @@ async function execDraftCreateApp(appName : string, pack? : string) : Promise<vo
     }
 }
 
-function draftCreateResult(sr : ShellResult, hadPack : boolean) {
+async function execDraftCreateDebug() {
```

Comment:
Can we deduplicate across this and `execDraftCreate`?  It seems to be the same code and UI except for passing the `node-debug` pack instead of `undefined` in the last line.